### ISSUE TITLE
No input found for String

### ISSUE THREAD

**Issue by mattclar** [plataformatec/simple_form#1381]
#So I just upgraded to rails 5 beta 4 and I am now getting the following error on any page which has any type of simple form, unfortunately even my login screen!!
I am using Jruby 9.1 also and postgres database.

```
RuntimeError - No input found for #<ActiveModel::Type::String:0x25846efd>:
simple_form (3.2.1) lib/simple_form/form_builder.rb:574:in `find_mapping'
simple_form (3.2.1) lib/simple_form/form_builder.rb:503:in `find_input'
simple_form (3.2.1) lib/simple_form/form_builder.rb:113:in `input'
app/views/appointments/_form.html.erb:36:in `block _app_views_appointments__form_html_erb___407106411_21258'
```

**Comment by kevin-klein** [comment]
I had the same issue, i used this monkey patch to fix it:

```ruby
module SimpleForm

  class FormBuilder

    # Attempt to guess the better input type given the defined options. By
    # default alwayls fallback to the user :as option, or to a :select when a
    # collection is given.
    def default_input_type(attribute_name, column, options)
      return options[:as].to_sym if options[:as]
      custom_type = find_custom_type(attribute_name.to_s) and return custom_type
      return :select             if options[:collection]

      input_type = column.try(:type)
      case input_type
      when :timestamp
        :datetime
      when ActiveModel::Type::Boolean
        :boolean
      when ActiveModel::Type::String, nil
        case attribute_name.to_s
        when /password/  then :password
        when /time_zone/ then :time_zone
        when /country/   then :country
        when /email/     then :email
        when /phone/     then :tel
        when /url/       then :url
        else
          file_method?(attribute_name) ? :file : :string
        end
      else
        input_type
      end
    end

  end

end
```

The problem comes from postgres column which returns :string for the the mri pg gem but an instance of ActiveModel::Type::String for activerecord-jdbcpostgresql-adapter.

**Comment by rafaelfranca** [comment]
Thanks! This is going to be fixed with https://github.com/plataformatec/simple_form/issues/1395

**Comment by herminiotorres** [comment]
@mattclar you could create a test application to reproduce this error and share where? I'm a bit in doubt of how to get to reproduce your error.

Thank you.

**Comment by lucasmazza** [comment]
Fixed by #1417 - please try out Simple Form `3.3.0` :)
